ci-operator/step-registry/cucushift/installer/check/cucushift-installer-check-commands.sh

```diff
@@ -1,8 +1,8 @@
 #!/bin/bash
 
 set -o nounset
-set -o errexit
-set -o pipefail
+# set -o errexit
```

Comment:
Let us copy the step to `cucushift-installer-check-print-info`, and include it in `cucushift-installer-check` chain.

Reply:
remove it from test phase?